### DISCUSSION TITLE
How to run Snaps in Firejail? [answer: you can't]

### DISCUSSION THREAD

**Discussion by Area51Kacz** [netblue30/firejail#4253] [Q&A] [answered]
How could i run snap apps  via firejail _(as we fun apps from  repositories / compilation / appimages)_

_why i want it...  arch aint has apparmor -  so i wanna  use firejail >)_
**Host... arch/manjaro**

**Comment by ghost** [comment, marked as answer]
Quoting from the firejail man:

Alternative sandbox technologies like `snap` (https://snapcraft.io/) and `flatpak` (https://flatpak.org/) are `not supported`. Snap and flatpak packages have their own native management tools and will not work when sandboxed with Firejail.

> why i want it... arch aint has apparmor - so i wanna use firejail >)
 Host... arch/manjaro

Not sure what you mean here. Arch Linux definately has [AppArmor](https://wiki.archlinux.org/title/AppArmor), and the firejail package from the [official repo](https://archlinux.org/packages/community/x86_64/firejail/) has it enabled. I'm not familiar with Manjaro but it would be surprising if they didn't do the same. You will have to configure and enable AppArmor though. Have you tried that yet? Consult the [firejail wiki](https://wiki.archlinux.org/title/Firejail) page for more details.
